vcd/writer.py

```diff
@@ -79,6 +79,7 @@ def __init__(self, file, timescale='1 us', date=None, comment='',
         self._scope_var_names = {}
         self._scope_types = {}
         self._ident_values = OrderedDict()
+        self._init_timestamp = init_timestamp
         self._prev_timestamp = 0
```

Comment:
There's another problem. in `register_var()` there is a call to `self.change(var, 0, init)` . If we call this with `_prev_timestamp` then we will fail the `timestamp < self._prev_timestamp` check inside `change()`